pkg/utils/docker.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+
+	"github.com/kubernetes-incubator/kompose/pkg/kobject"
+)
+
+func DockerBuildImage(name string, service kobject.ServiceConfig, composeFileDir string) string {
+	image := name
+	if service.Image != "" {
+		image = service.Image
+	}
+
+	cmd := NewCommand(fmt.Sprintf("docker build -t %s %s", image, service.Build))
```

Comment:
Using Docker API would be better, but it will need some more work. We are already executing system commands with ``os/exec`` anyways. However, I get your concern here. In this case, we are executing command based on user input, which can cause issues.

I don't like the native API for ``os/exec.Command`` to pipe commands. It's complicated. Will doing some validations around the user input solve this? For example, docker image name should not contain ``space`` or ``|``?

Is there some high level wrapper around os/exec in the golang world?